onnxruntime/core/providers/nnapi/nnapi_builtin/builders/shaper.cc

```diff
@@ -374,17 +377,10 @@ Status Shaper::SqueezeImpl(const std::string& input_name,
                            const std::string& output_name) {
   const Shape& input_dimen = shape_map_.at(input_name);
   int32_t input_size = input_dimen.size();
```

Comment:
Would be good to add a comment saying that 'axes' has been populated default values if the optional input to the ONNX operator was not provided.